src/vue-components/infl-footnote.vue

```diff
@@ -63,7 +66,39 @@
 
         target.setAttribute('data-x', x)
         target.setAttribute('data-y', y)
+      },
+
+      isOutOfRightXBound(childBR, parentBR) {
+        return (childBR.x + childBR.width) > (parentBR.x + parentBR.width)
+      },
+
+      isOutOfLeftXBound(childBR, parentBR) {
+        return childBR.x < 0
+      },
+
+      checkBounds () {
+        let inflpopup = this.$el.querySelector('.alpheios-inflections__footnote-popup')
+        let inflpanel = this.$el.closest('#alpheios-panel__inflections-panel')      
```

Comment:
Maybe we can cache references to `inflpopup` and `inflpanle` and reuse them for closing to avid extra queries?

Reply:
hmm. if we cache inflpopup, would this work if multiple footnotes get opened?